code/modules/mob/living/carbon/human/say.dm

```diff
@@ -169,6 +169,13 @@
 
 		if(span)
 			S.message = "<span class='[span]'>[S.message]</span>"
+
+	if(istype(wear_mask, /obj/item/clothing/mask/gas/voice_modulator))
```

Comment:
This is really snowflakey, and isn't acceptable--you're going to have to make this more OOP if this is remotely going to have a chance of passing.

Reply:
@Fox-McCloud absolutely, I thought the way this was implemented was weird too. I was following the example from the horse mask above and I didn't think I should deviate from the pattern already established.

However if you have examples or ideas for how you would prefer it to be coded I am very interested to learn. Thanks for taking a look!